### DISCUSSION TITLE
macOS homebrew 最新版本的 python 会内存溢出

### DISCUSSION THREAD

**Discussion by mark9804** [Arondight/Adachi-BOT#876] [General]
@Xm798 快跑

<img width="274" alt="image" src="https://user-images.githubusercontent.com/9006264/184519863-ac1a21f9-1f29-449c-9d25-22e731edf23f.png">
<img width="772" alt="image" src="https://user-images.githubusercontent.com/9006264/184519865-1beba0c3-201b-4a4d-bf5c-7ec804a05dbe.png">

**Comment by Arondight** [comment]
最新的是 py310 啊……

**Reply by mark9804** [reply]
homebrew 的主线还是 3.9.13，3.10 还是 keg-only

**Comment by Xm798** [comment]
啊这……
